spec/lib/local-links-manager/import/services_importer_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'rails_helper'
 require 'local-links-manager/import/services_importer'
+require 'services'
```

Comment:
Is this `require` statement needed here? We don't seem to do it for the Interactions specs